variables.tf

```diff
@@ -36,15 +36,33 @@ variable "pipeline_custom" {
   default     = null
 }
 
+variable "enable_nginx_ingress_metrics" {
```

Comment:
Maybe rename to `enable_metrics`? nginx_ingress is assumed given repo.

Reply:
Changedx